src/runtime/vendor/github.com/kata-containers/govmm/qemu/qemu.go

```diff
@@ -1830,7 +1830,7 @@ func (bridgeDev BridgeDevice) QemuParams(config *Config) []string {
 		deviceParams = append(deviceParams, fmt.Sprintf("%s,bus=%s,id=%s", driver, bridgeDev.Bus, bridgeDev.ID))
 	default:
 		driver = PCIBridgeDriver
-		shpc := "off"
+		shpc := "off,io-reserve=4k,mem-reserve=1m,pref64-reserve=1m"
```

Comment:
nice, this is the trick to make it works ... I guess you will document this since I see magic numbers (4k, 1m) , I wonder if these sizes depend on the workload and device hot plugged (gpus, nets, etc) :thinking:

Reply:
Right, this is just a hacked up version of the change to test if it works (it shouldn't sit in govmm itself at all).  I'll be working on a cleaned up version of this fix shortly.